src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -458,7 +458,7 @@ private string GenerateRow(string[] values, ReadOnlySpan<int> alignment, Display
                 }
 
                 sb.Append(GenerateRowField(values[k], _si.columnInfo[k].width, alignment[k], dc, addPadding));
-                if (values[k].Contains(ESC))
+                if (values[k] is not null && values[k].Contains(ESC))
```

Comment:
```suggestion
                if (values[k]?.Contains(ESC))
```

Reply:
> if (values[k]?.Contains(ESC))

It is not work (can return null but bool is expected..

Reply:
Do I need to do anything here? Sorry, a bit new to this.

Reply:
Nah, you're fine. I misremembered how these operators work a little bit.